docs/Contributing.md

```diff
@@ -60,6 +60,20 @@ npm start
 open http://localhost:8000/
 ```
 
+### Static Docs Site
+
+Markdown and source code for the [static docs site](https://pricelinelabs.github.io/design-system/) is located in
+`docs/`. This [x0](https://www.npmjs.com/package/@compositor/x0) project requires Node 7+.
+
+To run the static docs locally:
+
+```sh
+cd docs
+npm install -g @compositor/x0
```

Comment:
This step shouldn't be necessary since this is installed locally

Reply:
Good call; looks like just running `npm i` is sufficient.